src/cloud_tasks/cli.py

```
@@ -626,252 +470,885 @@ async def list_running_instances_cmd(args: argparse.Namespace, config: Config) -
+                task_file=args.task_file,
+                start_task=args.start_task,
+                limit=args.limit,
+                max_concurrent_queue_operations=args.max_concurrent_queue_operations,
+                force=getattr(args, "force", False),
+            )
+
+        # Create orchestrator (disable auto-termination since we control completion via SQLite)
+        logger.info("Creating instance orchestrator...")
+
+        # Provide callback to get remaining task count from database
+        def get_remaining_task_count():
```

Ruff (0.14.14)

[warning] Missing return type annotation for private function `get_remaining_task_count`

(ANN202)